src/server/rdb_test.cc

```diff
@@ -158,7 +158,7 @@ TEST_F(RdbTest, Stream) {
 
   resp = Run({"xinfo", "groups", "key:1"});  // test dereferences array of size 1
   EXPECT_THAT(resp, ArrLen(8));
-  EXPECT_THAT(resp.GetVec(), ElementsAre("name", "g2", "consumers", "0", "pending", "0",
+  EXPECT_THAT(resp.GetVec(), ElementsAre("name", "g2", "consumers", IntArg(0), "pending", IntArg(0),
                                          "last-delivered-id", "1655444851523-1"));
```

Comment:
Just a minor nit for the future :)
You could use something like:
```cpp
EXPECT_THAT(resp, RespArray(ElementsAre(...)));
```

This has the benefits of:
1. Failing the test if `resp` is not an array response (instead of crashing)
2. Not needing to check `ArrLen(8)` (we usually did this to make sure that the response is an array), as `ElementsAre()` verifies the count as well as the content
3. It supports hierarchical arrays, like `RespArray(ElementsAre(..., RespArray(ElementsAre(...)), ...)`